etsin_finder/frontend/js/components/qvain/index.jsx

```diff
@@ -165,11 +185,19 @@ class Qvain extends Component {
           .catch(this.handlePublishError)
       })
       .catch(err => {
+        console.log('Error for event: ', e)
         console.log(err.errors)
 
         // Refreshing error header
-        this.setState({ response: null })
-        this.setState({ response: err.errors })
+        this.setState({
+          response: null
+        })
```

Comment:
suggestion, set response to null in the beginning of the method
```suggestion
        
```

Reply:
There is nothing wrong with the implementation but just thought of a "possibly" cleaner way to set the value

Reply:
Good suggestion as well, should be resolved with commit 5bf17c4